.scalafmt.conf

```diff
@@ -8,5 +9,6 @@ optIn.annotationNewlines = true
 project.excludeFilters = [
   scalafix-tests/input
   scalafix-tests/output
+  scalafix-tests/shared/src/main/scala/test/PrettyTest.scala
```

Comment:
Why did this file get excluded?

Reply:
Since `PrettyTest.scala` is used only for testing the 
pretty-printer of scalafix https://github.com/scalacenter/scalafix/blob/6db013fdef8fcb8070cf183881071f9ee3fe1948/scalafix-tests/unit/src/test/scala/scalafix/tests/util/PrettyExpectSuite.scala#L11-L40.
(comparing it with [Pretty.expect](https://github.com/scalacenter/scalafix/blob/1dec92e78784016b35fc44d0efee230198145648/scalafix-tests/unit/src/main/resources/expect/Pretty.expect)), scalafmt shouldn't format the `PrettyTest.scala` to avoid test failure.

Reply:
Aaah, makes sense 👍